### DISCUSSION TITLE
Scrolling Menus

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#411] [Ideas]
The system menu will be a little bit different on the small screen, but I have an idea as shown in the picture.
![think](https://user-images.githubusercontent.com/44593430/204142365-de6ccb7b-ba4f-4663-879b-ab96dc94f65b.png)

**Comment by probonopd** [comment]
helloSystem is for desktop computers, so I'd consider 800x600 a system requirement, maybe 640x480 if you are reall desparate, but no less.

**Reply by louies0623** [reply]
Apart from the fact that my screen is too small, the main point is that the menus of some software are sometimes very long and need to be scrolled on the top menu.

**Reply by probonopd** [reply]
Well, that is not an issue with screens of the recommended size. For desktops, I would say FullHD is the minimum standard nowadays.

**Reply by louies0623** [reply]
The resolution of this machine is1024x600, 
But sometimes the menu is too full to show only half.

![image](https://user-images.githubusercontent.com/44593430/209589630-9f5a35b3-5f46-4ea8-a0fd-72b03f839405.png)

**Reply by louies0623** [reply]
![VirtualBox_PC-2_29_12_2022_01_02_22](https://user-images.githubusercontent.com/44593430/209847008-b2366baa-1fb7-435c-add5-1cb920e87754.png)

**Reply by louies0623** [reply]
Mac OS 8 

https://user-images.githubusercontent.com/44593430/209907389-3d1d135b-9984-483b-82a2-f282d8d4aad2.mp4

**Comment by louies0623** [comment]
I want it to support raspberry pi screen, as shown in the picture.
![image](https://user-images.githubusercontent.com/44593430/204713887-a711c6db-0ddb-4361-a0e0-156c001f2811.png)

**Reply by probonopd** [reply]
That's a neat little display but it's more suitable for a mobile user interface.
helloDesktop, as the name implies, is for desktop systems.

**Reply by louies0623** [reply]
Maybe one day someone will use HelloSystem to install this installation for developing BSD software.

**Reply by probonopd** [reply]
You mean: The Raspberry Pi screen as an _additional_ screen, in addition to a big one?

**Reply by louies0623** [reply]
Right.

**Reply by probonopd** [reply]
That should be doable. Main screen shows the Menu. The small screen could show additional desktop/window.

**Reply by louies0623** [reply]
But there is one thing, if he does not have a screen to carry, only this 3.5-inch thing, it still needs to work, for example: network line operator.

**Comment by jjuran** [comment]
If I had that screen in addition to a main one, I'd want it as its own display, not part of the desktop.  As a solitary portable screen, I'd want a touch interface.  YMMV.

**Comment by louies0623** [comment]
![VirtualBox_PC-1_01_12_2022_21_00_13](https://user-images.githubusercontent.com/44593430/205059578-3482edaf-91d1-43c0-b37d-b7c9a36f52f7.png)
I have noticed that if the list is too long as shown in the figure, it can be changed to the following method if possible.
![think](https://user-images.githubusercontent.com/44593430/205097387-5c203f63-825a-455a-86a7-a2cec697b116.png)

**Comment by louies0623** [comment]
@probonopd I found that the screens in some workplaces are vertical, but the menus of this system are horizontal, and there will be display problems, such as computers in hospitals and recording studios, as shown in the picture.

![image](https://user-images.githubusercontent.com/44593430/208876351-03258d26-9d21-497c-8606-c52884c81ec2.png)

![image](https://user-images.githubusercontent.com/44593430/208873037-a15890c5-fa2f-4254-b5bf-977312a62468.png)

**Reply by probonopd** [reply]
Yes. We are aiming at a traditional desktop computer setup with one main screen and possibly additional auxiliary screens.

**Comment by louies0623** [comment]
@probonopd In Macintosh human interface guidelines pdf 103/418
it is clearly written.
